src/main/java/uk/gov/companieshouse/api/testdata/repository/UserRepository.java

```diff
@@ -0,0 +1,12 @@
+package uk.gov.companieshouse.api.testdata.repository;
+
+import org.springframework.data.mongodb.repository.MongoRepository;
+import org.springframework.data.repository.NoRepositoryBean;
+import uk.gov.companieshouse.api.testdata.model.entity.Users;
+
+/**
```

Comment:
Do we need the comment Repository for the Users collection ?  It's not on the repository class above